src/authkit-callback-route.ts

```diff
@@ -57,6 +69,16 @@ export function authLoader(options: HandleAuthOptions = {}) {
         session.set('jwt', encryptedSession);
         const cookie = await commitSession(session);
 
+        if (onSuccess) {
+          await onSuccess({
```

Comment:
We should add an example of how to use this method (and what it provides) in the readme.